src/components/camera-tool.js

```diff
@@ -195,12 +194,14 @@ AFRAME.registerComponent("camera-tool", {
       this.screen = new THREE.Mesh(geometry, material);
       this.screen.rotation.set(0, Math.PI, 0);
       this.screen.position.set(0, 0, -0.042);
+      this.screen.onBeforeRender = onBeforeRender;
       this.screen.matrixNeedsUpdate = true;
       this.el.setObject3D("screen", this.screen);
 
       this.selfieScreen = new THREE.Mesh(geometry, material);
       this.selfieScreen.position.set(0, 0.4, 0);
       this.selfieScreen.scale.set(-2, 2, 2);
+      this.selfieScreen.onBeforeRender = onBeforeRender;
```

Comment:
Hmm don't remember why we didn't do this originally... I bet we were doing the actual rendering work in the `update` callback before and we didn't want to do it twice so we applied it to the material instead of the object. This definitely makes more sense now though given its only setting a flag anyway.